css/sponsor-dashboard.css

```diff
@@ -90,17 +89,16 @@ table.sponsor-list-table td{
     border-top: 1px solid rgb(195, 190, 190);
     border-collapse: collapse;
 }
-.status-active{
-    background: #79D733;
+.status{
     border-radius: 10px;
     text-align: center;
     padding: 5px 20px 5px 20px;
 }
-.status-deactive{
-    background: #BECAD6;
-    border-radius: 10px;
-    text-align: center;
-    padding: 5px 20px 5px 20px;
+.active{
```

Comment:
We should scope these with status as this is only concerned with status.
like `.status.active`